### DISCUSSION TITLE
Saving passwords

### DISCUSSION THREAD

**Discussion by gadefox** [qutebrowser/qutebrowser#6702] [Q&A]
I can't find any quick guide how to configure this feature.

**Comment by toofar** [comment]
It's not a feature, use a dedicated password manager! There are userscripts to integrate several of them with qutebrowser https://github.com/qutebrowser/qutebrowser/tree/master/misc/userscripts
See the docs for :spawn for launching userscripts (and you can bind it to keys) https://qutebrowser.org/doc/help/commands.html#spawn

**Reply by gadefox** [reply]
Okay, mea culpa, it's a dedicated pwd mngr, not a feature.. And sorry for a noob question : which one is the best choice on Ubuntu/Debian? Also pls understand I don't have time for studying other scripts.. I just need a feature ASAP, thanks :)..

**Reply by toofar** [reply]
Sorry, I don't have an answer to that, it's too subjective. pass and keepassxc are probably the most popular.

**Reply by gadefox** [reply]
You're right, this is too subjective. On the other side, other users in my shoes can save the time and follow my advice. My choice is based on following parameters:
- I'm a minimalist : With respect, but 20+ MB solutions for simple tasks e.g executing `gpg` and creating files in folders.. Are they joking??
- I prefer text script before binary versions, because I can review what it's doing.
- I prefer highly configurable solutions, simple : for example I'm happy w/ pretty old `xpaint` (900kB) instead of `pinta` (150+MB due to Mono runtime).

Therefore I go for 
- a simple shell script [pass](https://www.passwordstore.org/)
- with [dmenu](https://tools.suckless.org/dmenu/)
- and my [version](https://github.com/gadefox/etc/blob/main/qutebrowser/pass) of `qute_pass` userscript from yr list.
